src/frequenz/sdk/microgrid/_battery/_status.py

```diff
@@ -113,12 +123,10 @@ def is_blocked(self) -> bool:
         return self.blocked_until > datetime.now(tz=timezone.utc)
 
 
-class BatteryStatusTracker(AsyncConstructible):
+class BatteryStatusTracker:
```

Comment:
This too,  I think should sit outside the `microgrid` package.  The `microgrid` package is supposed to be a thin wrapper providing access to data from the microgrid and for sending commands to the microgrid.

But this class is really stateful tracking logic, which should ideally be part of a higher level package,  possibly the `actor/power_distributing`.   I see that this code was already merged,  so I can move out later also, if you prefer.

Reply:
Moved to `actor/power_distributing`